### DISCUSSION TITLE
Storyboards with many background elements are very choppy.

### DISCUSSION THREAD

**Discussion by NatureHog** [ppy/osu#17620] [Q&A] [answered]
I don't know how to phrase it better than in the title. It only happens when storyboard is enabled, and there are a lot of storyboard elements to be seen.

I already made [a video](https://1drv.ms/u/s!AqwZ15G5dYM5gswLbIvydL44bxxeNg?e=Nmm3Cb) of the problem, with three different beatmaps with storyboards, and the debug overlay shown.

Video notes:
First beatmap begins at around 30sec, choppiness begins at ~ 3min, at ~ 4min 10sec beatmap is repeated w/o storyboard with no noticable choppiness. 
Second beatmap begins w/o storyboard at ~9min 15sec with no noticeable choppiness, beatmap is repeated with storyboard at ~13min 20sec, Update graph gets higher at ~14min 31sec with no noticeable choppiness, it gets more choppy at ~15min 20sec.
Third beatmap begins at ~17min 20sec with no noticeable choppiness at all.

OS is Windows 11 64bit, Multithreading in osu!lazer is enabled (idk if that makes a difference...)

**Comment by Theighlin** [comment, marked as answer]
#3756
